package/firmware/as21x1x-firmware/Makefile

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=as21x1x-firmware
```

Comment:
> Add support for Aeonsemi AS21xxx firmware blob. Firmware has been
submitted and accepted to linux-firmware. Current version is 1.8.2.

Why not take it from linux-firmware directly. OpenWrt linux-firmware package is at 20250808 which should include this firmware